packages/user-interface/src/components/CasesList.tsx

```diff
@@ -45,7 +43,7 @@ const CasesList = ({
   const { paths } = useOutletContext<RouterOutletContext>();
   const casesPath = readMoreLink || paths.cases;
   const listView = Boolean(readMoreAmount && readMoreAmount > 8);
-  const title = showTitle
+  const title = titleTranslationId
     ? intl.formatMessage({ id: titleTranslationId })
     : undefined;
```

Comment:
Wanneer titleTranslationId string | undefined is, zou je hier dit kunnen doen:

`const title = titleTranslationId 
  && intl.formatMessage({ id: titleTranslationId });`